x-pack/plugins/cloud_security_posture/public/pages/compliance_dashboard/compliance_dashboard.tsx

```diff
@@ -332,6 +332,7 @@ export const ComplianceDashboard = () => {
   return (
     <CloudPosturePage>
       <EuiPageHeader
+        data-test-subj={'cloud-posture-dashboard-page-header'}
```

Comment:
maybe add this to test_subjects.ts as well ? to make it similar to the other data-test-subj added on this PR

Reply:
💯%, ty for catching that